projects.py

```diff
@@ -59,7 +63,56 @@ def run_projects(c, project_name=None):
             run_project(c, timestamp, project_name)
         else:
             for project_name in c.load_schema().keys():
-                run_project(c, timestamp, project_name)
+                if project_name.startswith("exact:"):
+                    run_exact_match(c, timestamp, project_name)
+                else:
+                    run_project(c, timestamp, project_name)
+
+
+def run_exact_match(c, metadata_timestamp, project_name=None, households=False):
+    with open(
+        Path(c.project_results_dir) / f"project-metadata-{metadata_timestamp}.json", "r"
+    ) as metadata_file:
+        metadata = json.load(metadata_file)
+    project_start_time = datetime.datetime.now()
+    metadata["projects"].append(project_name)
+    metadata[project_name] = {"start_time": project_start_time.isoformat()}
+    clks_set = {}
+    clks = {}
+    result_json = {"groups": []}
+    project = project_name[len("exact:") :]
+    for system in c.systems:
+        raw_clks = None
+        if households:
+            raw_clks = c.get_household_clks_raw(system, project)
+        else:
+            raw_clks = c.get_clks_raw(system, project)
+        clk_json = json.loads(raw_clks)
+        clks_set[system] = set(clk_json["clks"])
+        clks[system] = clk_json["clks"]
+
+    for pair in itertools.combinations(c.systems, 2):
```

Comment:
@jsrockhill Overall the approach looks good, but I notice this only looks at pairwise matches, not matches across all size combinations (ex, no 4-way exact matches)
Conceptually I think the pairwise approach should be equivalent to looking at all sizes (if we know that A:1 and B:2 are an exact pair, and B:2 and C:4 are an exact pair, we know that A:1 B:2 C:4 is necessarily an exact triple) but I'm not sure offhand if the way we handle deconfliction could potentially produce different results depending on the approach. What do you think?

Note we have an example of doing n-wise matching in concordance.py: https://github.com/mitre/linkage-agent-tools/blob/master/dcctools/concordance.py#L52